ragen/llm_agent/es_manager.py

```
@@ -306,13 +322,23 @@ def get_rollout_states(self):
             for k, v in custom_metric.items():
                 # TODO: Move TURN_LVL_METRICS into the environment
                 if "webshop" not in cache['tag'].lower() or ("webshop" in cache['tag'].lower() and k in TURN_LVL_METRICS):
                     env_metric[k] = np.sum(v) / (len(cache['history']) - 1) # NOTE: exclude the last observation
                 else:
                     env_metric['traj_sum/' + k] = np.sum(v)
-
+            try:
+                if 'score' in custom_metric and len(custom_metric['score']) > 0:
+                    env_metric['final_score'] = float(custom_metric['score'][-1])
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)